views/js/buckaroo.js

```diff
@@ -175,6 +175,28 @@ function buckaroo($) {
                 }
             }
         },
+        requiredIssuers: (element, errorLabel, errorMessage) => {
+            let radioInputs = $(`.${element}:input[type="radio"]`);
+            let select = $(`.${element}`);
+
+            if (radioInputs.length !== 0){
+                let inputChecked = radioInputs.filter(':checked');
+
+                if (inputChecked.length === 0) {
+                    methodValidator.valid = false;
+                    methodValidator.displayMessage($(errorLabel), errorMessage, false);
+                }
+                return;
+            }
+
+            if (select.length !== 0) {
+                let selectedOption = select.find('option:selected');
+                if (selectedOption.length === 0 || selectedOption.val() === '0') {
+                    methodValidator.valid = false;
+                    methodValidator.displayMessage($(errorLabel), errorMessage, false);
+                }
+            }
```

Comment:
better if we split this into 2 functions, one for validating radio groups and one for validating select, also you can reduce the checks to `$(`.${element}:input[type="radio"]:checked`).length > 0;` and for the select `$(`.${element}`).val().length > 0;` or similar